pkg/config/config.go

```diff
@@ -92,6 +94,13 @@ func (c *config) addKeys() {
 
 // Process does the bulk of the math reading the CSV raw data and saving results
 func (c *config) Process() {
+
+	var m []metrics.Metrics
+	err := utils.GetMetics(c.searchDir, &m)
```

Comment:
I'd rather have it get misc metrics after the primary CSV processing, as this is secondary.

Reply:
Done!